### DISCUSSION TITLE
Multi-Device Interaction Model

### DISCUSSION THREAD

**Discussion by robot-ranger** [mtconnect/cppagent#390] [General]
We are highly interested in the Task model shown under the 'Multi-Device Interaction Model' in the latest SysML:

https://model.mtconnect.org/#Package__8ee3ad7b-5e65-4031-a849-a64f6a3b2fc2

Is this functioning yet? I cant quite tell from the SysML. I can obviously see that it isnt in the released standard.
Where can we get involved in the discussion on this? (or maybe we can start a discussion here?)
We have a 'little black box' for MTC Read/Read interfaces and we are scoping up a tasking model to enable mobile robots for machine tending applications. We will pursue our own model, but given the opportunity, we would rather contribute to a common model!

**Comment by robot-ranger** [comment]
@wsobel - how can we get involved in multi device interaction?

**Reply by wsobel** [reply]
Give me a few days to get more information on this. There may be a project starting up and we can discuss involvement. 

Please direct message me at: will@metalogi.io and we can discuss.

Thanks,
- W
